tests/test_packaging.py

```diff
@@ -199,26 +201,25 @@ def test_polish_and_ratchet_pins_and_upper_bounds():
         ("requirements-dev-lite.txt", "snapshottest", (("==", "0.6.0"),)),
         ("requirements-dev-mysql.txt", "PyMySQL", (("<", "0.10"), (">=", "0.9.3"))),
         ("requirements-dev-pagerduty.txt", "pypd", (("==", "1.1.0"),)),
-        (
-            "requirements-dev-sqlalchemy-less-than-2.txt",
-            "sqlalchemy",
-            (("<", "2.0.0"),),
-        ),
         (
             "requirements-dev-sqlalchemy.txt",
             "PyMySQL",
             (("<", "0.10"), (">=", "0.9.3")),
         ),
         ("requirements-dev-sqlalchemy.txt", "ipykernel", (("<=", "6.17.1"),)),
         ("requirements-dev-sqlalchemy.txt", "moto", (("<", "3.0.0"), (">=", "2.0.0"))),
+        ("requirements-dev-sqlalchemy.txt", "pandas", (("<", "2.0.0"),)),
```

Comment:
Wanting to check: should these be removed since `sqlalchemy < 2` is handled by the `requirements-dev-sqlalchemy1.txt` file?